ocaml/idl/datamodel_host.ml

```diff
@@ -1766,6 +1766,13 @@ let apply_updates =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let rescan_drivers =
```

Comment:
so the reason to put `rescan_driver` in the host class is that it can only be called on hosts. How about `select` and `deselect`, can they only be called on hosts?

Reply:
The driver uuid already encodes which host it's located on; specifying both the host uuid and the driver uuid would be redundant - the call already has the information necessary to determine where to forward it.